src/vs/workbench/parts/lib/node/configVariables.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as Types from 'vs/base/common/types';
+import { SystemVariables } from './systemVariables';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { IWorkbenchEditorService } from 'vs/workbench/services/editor/common/editorService';
+import { IWorkspaceContextService } from 'vs/workbench/services/workspace/common/contextService';
+import URI from 'vs/base/common/uri';
+
+export class ConfigVariables extends SystemVariables {
+	constructor(private configurationService: IConfigurationService, editorService: IWorkbenchEditorService, contextService: IWorkspaceContextService, workspaceRoot: URI = null, envVariables: { [key: string]: string } = process.env) {
+		super(editorService, contextService, workspaceRoot, envVariables);
+	}
+
+	protected resolveString(value: string): string {
+		value = super.resolveString(value);
+
```

Comment:
The regexp should have 'config' not 'settings' to allign with other places where we have replacments.